lib/engine-jit/src/artifact.rs

```diff
@@ -141,21 +137,26 @@ impl JITArtifact {
     }
 
     /// Deserialize a JITArtifact
-    pub fn deserialize(jit: &JITEngine, bytes: &[u8]) -> Result<Self, DeserializeError> {
+    ///
+    /// # Safety
+    /// This function is unsafe because rkyv reads directly without validating
+    /// the data.
+    pub unsafe fn deserialize(jit: &JITEngine, bytes: &[u8]) -> Result<Self, DeserializeError> {
         if !Self::is_deserializable(bytes) {
             return Err(DeserializeError::Incompatible(
                 "The provided bytes are not wasmer-jit".to_string(),
             ));
         }
 
-        let inner_bytes = &bytes[Self::MAGIC_HEADER.len()..];
-
-        // let r = flexbuffers::Reader::get_root(bytes).map_err(|e| DeserializeError::CorruptedBinary(format!("{:?}", e)))?;
-        // let serializable = SerializableModule::deserialize(r).map_err(|e| DeserializeError::CorruptedBinary(format!("{:?}", e)))?;
+        let mut inner_bytes = &bytes[16..];
```

Comment:
Goes for here too, and just in general... This can be done as a follow up to this PR if you really want to ship it, but for the long term health of this code, using named constants can be really helpful

Reply:
Fixed!